gdal-sys/src/lib.rs

```diff
@@ -3,5 +3,11 @@
 #![allow(non_snake_case)]
 #![allow(clippy::upper_case_acronyms)]
 #![allow(rustdoc::bare_urls)]
+// bindgen test code generates lots of warnings when testing
+// sizes of types due to using constructs such as:
+// `unsafe { &(*(::std::ptr::null::<__sbuf>()))._base as *const _ as usize }`
+// This disables those warnings.
+#![cfg_attr(test, allow(deref_nullptr))]
```

Comment:
I think we should upgrade `bindgen` instead. This was probably fixed back in May, in https://github.com/rust-lang/rust-bindgen/pull/2203.